webui/react/src/types.ts

```diff
@@ -10,6 +10,11 @@ export interface Auth {
   user?: User;
 }
 
+export interface Credentials {
+  username: string;
+  password: string;
```

Comment:
thought: should we sort interface and type members alphabetically?

Reply:
yeah ideally we get eslint to format and check it.

Reply:
updated the order. and we discussed the eslint rule. coming up in a separate PR.